### ISSUE TITLE
actual kernel version???

### ISSUE THREAD

**Issue by hoh61** [ubports/ubuntu-touch#645]
### Illustrations

- Device: BQ M10 FHD frieza
- Channel: 16.04/devel
- Build: r88

### Steps to reproduce

uname -a

### Expected behavior

kernel version of 16.04 (somewhat 4.4 or higher)

### Actual behavior

Linux ubuntu-phablet 3.10.93+ #1 SMP PREEMPT Sun Sep 24 04:28:52 CEST 2017 aarch64 aarch64 aarch64 GNU/Linux

### Logfiles and additional information

cat /proc/version is giving the following output:
Linux version 3.10.93+ (mariogrip@mariogrip-laptop) (gcc version 4.9.x-google 20140827 (prerelease) (GCC) ) #1 SMP PREEMPT Sun Sep 24 04:28:52 CEST 2017

Which version of Kernel is really used? this version is giving some intermediate ubuntu version between 13.04 and 13.10 !!!

I wanted to compile the missing usb kernel modules!

**Comment by hoh61** [comment]
As a consequence, we do not use 16.04 linux kernel???

And yes, I do not understand Android. It is said it is just a linux derivate, open source with a little closed source...

**Comment by Flohack74** [comment]
Yes, this is the consequence. While Android is using Linux kernel, it is not Linux, otherwise it would be much easier to get Ubuntu Touch working. We can only use the kernel that actually is compatible with the original manufacturers userland blobs / binaries. Because they are doing the heavy lifting to get the small shim of open sourced drivers (some of them are more like system hooks for userland blobs) working. Remember no Android manufacturer really wants open drivers, they can hide a lot of IP plus probably stolen code from other vendors plus crappy coding behind this :)

I closed this issue as it is really not possible, not even lets say to go from kernel 3.0 to 3.1 - yes some of my porting devices run a kernel that is that old.
